src/react-apps/applications/service-development/src/App.tsx

```diff
@@ -40,45 +40,143 @@ class AppClass extends React.Component<IAppProps, any> {
         path: '/om',
```

Comment:
Should all elements have English names?

Reply:
I thought it would be standard and consistent to have the english names as other parts of the application are using names in english

Reply:
@matsgm yes, i will make the change in the path as part of this pull request